launch/launch/actions/unset_environment_variable.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2018 Open Source Robotics Foundation, Inc.
```

Comment:
2019 (would use GitHub's suggestion feature, but mess with DCO)

Reply:
Updated the year.